branchinfo.txt

```diff
@@ -4,6 +4,6 @@
 MAJOR_VERSION=1
 MINOR_VERSION=1
 PATCH_VERSION=0
-RELEASE_SUFFIX=beta
-CHANNEL=master
-BRANCH_NAME=master
+RELEASE_SUFFIX=preview1
+CHANNEL=preview
+BRANCH_NAME=release/1.1.0
```

Comment:
@gkhanna79 Yes, just a FYI that auto-update depends on this config change and that it looks good. (I don't think I have have enough core-setup experience to review the rest of the PR well.)